### PR TITLE
Add MeetingBar cask (1.0b14)

### PR DESCRIPTION
**Pull request by leits**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/meetingbar.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  sha256 'fbc449bbb0996efb34eb4eae28e356e8ac2e6313b2c7190d6f7a9dcbfe0dad39'
+
+  appcast 'https://github.com/leits/MeetingBar/releases.atom'
+
+  url "https://github.com/leits/MeetingBar/releases/download/v#{version}/MeetingBar.dmg"
+  name 'MeetingBar'
+  homepage 'https://github.com/leits/MeetingBar'
+
+  app 'MeetingBar.app'
+end
```
Comment (ran-dall) [inline Casks/meetingbar.rb:12]:
```suggestion
  url "https://github.com/leits/MeetingBar/releases/download/v#{version}/MeetingBar.dmg"
  appcast 'https://github.com/leits/MeetingBar/releases.atom'
  name 'MeetingBar'
  homepage 'https://github.com/leits/MeetingBar'

  depends_on macos: '>= :catalina'

  app 'MeetingBar.app'
end

**Review by ran-dall — CHANGES_REQUESTED** [review]
@leitsius @leits I fixed your submission, however, we still may be missing the `zap` stanza. I don't have `10.15` installed, so there isn't a way for me to investigate. You may want to look into this is your app has folder/files that need to be deleted upon uninstalling. You can also look at @nrlquaker's [`createzap` tool](https://github.com/nrlquaker/homebrew-createzap) to make this easy on you.

**Comment by ran-dall** [comment]
@leitsius @leits  Did you check if there's a `zap` stanza that should be added?

**Comment by nrlquaker** [comment]
Please use the following zap:
```
zap trash: [
             '~/Library/Application Scripts/leits.MeetingBar',
             '~/Library/Containers/leits.MeetingBar',
           ]
```

**Comment by leits** [comment]
@ran-dall @nrlquaker Thank you very much!

**Review by ran-dall — APPROVED** [review]

**Pull request merged**